src/test/ui/const-generics/const_evaluatable_checked/from-sig-fail.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0080]: evaluation of constant value failed
+error[E0080]: evaluation of `test::<0_usize>::{constant#0}` failed
```

Comment:
we already report such paths (with {constant#0}) in other situations. It's not ideal, but maybe better than not mentioning the path?

Reply:
For reference, there are indeed a few cases in the ui tests where this happens today, for example https://github.com/rust-lang/rust/blob/ce0d64e03ef9875e0935bb60e989542b7ec29579/src/test/ui/mir/issue-80742.stderr#L10-L14 

(After/if this PR is merged, I'll work on changing these messages: to keep the originating span, but not the full instance path, so that they are not duplicated)